packages/react-components/react-toast/src/components/Toaster.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * ⚠️ This is temporary and WILL be removed
+ */
+
+import * as React from 'react';
+import { Portal } from '@fluentui/react-portal';
+import { useToaster, getPositionStyles } from '../state';
+import { Toast } from './Toast';
+import { makeStyles, mergeClasses } from '@griffel/react';
+
+const useStyles = makeStyles({
+  container: {
+    position: 'fixed',
+  },
+});
+
+export const Toaster: React.FC = () => {
+  const { getToastToRender, isToastVisible } = useToaster();
```

Comment:
```suggestion
  const { getToastsToRender, isToastVisible } = useToaster();
```

If it's a collection of toast....